Calendar/Calendar/CalendarForm.cs

```diff
@@ -26,13 +29,20 @@ public CalendarForm()
             _calendarData = GetDataProvider();
             InitializeDaysControls();
             UpdateCalendarView();
-            DisplayCurrentMonth();
+            UpdateCurrentMonthText();
         }
-
+        /// <summary>
+        /// The inferface ICalendarEventData creates new class InMemoryCalendarEventData
```

Comment:
Prawie dobrze każdy widzi że zwracany jest interfejs i że tworzony jest nowy obiekt InMemoryCalendarEventData. Ale co to znaczy tak naprawdę? Że zwracamy obiekt dostarczający nam Eventy. 
np Returns Event data provider for the calendar view.